altair/vegalite/v5/schema/channels.py

```diff
@@ -31189,3 +31189,48 @@ def encode(
         encoding.update(kwargs)
         copy.encoding = core.FacetedEncoding(**encoding)
         return copy
+
+
+class _EncodeKwds(TypedDict, total=False):
+    """Placeholder (FIXME)."""
+
+    angle: str | Angle | Map | AngleDatum | AngleValue
```

Comment:
I think the docstring here should have the parameter descriptions, but **not** the types.

This won't be going through `sphinx` so there is no need for the duplication IMO.

Also it will fit in beter with https://github.com/MarcoGorelli/polars/blob/db6d8f76782aa02bb72b923cf46e2f638ede6bef/py-polars/polars/dataframe/plotting.py#L54-L65

Reply:
Agree. Working on this now.

Reply:
[2b9ad2c](https://github.com/vega/altair/pull/3515/commits/2b9ad2cd74cf7de66655304be57214acd9e83758)

Reply:
> [2b9ad2c](https://github.com/vega/altair/pull/3515/commits/2b9ad2cd74cf7de66655304be57214acd9e83758) @dangotbanned thoughts?

I was writing my [comment](https://github.com/vega/altair/pull/3515/commits/2b9ad2cd74cf7de66655304be57214acd9e83758#r1703209502) at the same time as this apologies